### PR TITLE
Convert ilsgateway off of couch locations

### PR DESCRIPTION
**Pull request by esoergel**
:tropical_fish: 
@orangejenny

### CODE REVIEW

File: custom/ilsgateway/slab/views.py [hunk 1]
```diff
@@ -85,20 +80,20 @@ def post(self, request, *args, **kwargs):
...
                 )
                 slab_config.save()
         else:
-            SLABConfig.objects.filter(sql_location=self.sql_location).delete()
+            SLABConfig.objects.filter(sql_location=self.location).delete()
 
         messages.success(request, 'Location updated successfully')
         return HttpResponseRedirect(
-            reverse('slab_edit_location', kwargs={'domain': self.domain, 'location_id': self.location.get_id}),
+            reverse('slab_edit_location', kwargs={'domain': self.domain, 'location_id': self.location.location_id}),
```
Comment (dimagimon) [inline custom/ilsgateway/slab/views.py]: E501 line too long (116 > 115 characters)

**Comment by esoergel** [comment]
opening for tests - there's one that's failing locally also on master

File: custom/ilsgateway/tanzania/warehouse/updater.py [hunk 2]
```diff
@@ -209,28 +209,20 @@ def default_start_date():
...
-        returns test region and all its children
-    """
-    test_region = SQLLocation.objects.get(domain=domain, external_id=const.TEST_REGION_ID)
-    sql_locations = SQLLocation.objects.filter(
-        Q(domain=domain) & (Q(parent=test_region) | Q(parent__parent=test_region))
-    ).exclude(is_archived=True).order_by('id').only('location_id')
-    return [sql_location.couch_location for sql_location in sql_locations] + \
-           [test_region.couch_location]
+def _get_locations_by_type(type_name):
+    return list(SQLLocation.active_objects.filter(domain=domain, location_type__name=type_name))
```
Comment (dimagimon) [inline custom/ilsgateway/tanzania/warehouse/updater.py]: F821 undefined name 'domain'

Comment (esoergel) [inline custom/ilsgateway/tanzania/warehouse/updater.py]: fixed in a later commit

**Review by gcapalbo — APPROVED** [review]

**Pull request merged**